lib/abr/simple_abr_manager.js

```diff
@@ -345,8 +348,19 @@ shaka.abr.SimpleAbrManager = class {
    */
   getBandwidthEstimate() {
     const defaultBandwidthEstimate = this.getDefaultBandwidth_();
-    const bandwidthEstimate = this.bandwidthEstimator_.getBandwidthEstimate(
+    let bandwidthEstimate = this.bandwidthEstimator_.getBandwidthEstimate(
         defaultBandwidthEstimate);
+
+    // Return the mini(bandwidthEstimate, the network API estimate)
```

Comment:
This description is very confusing. I'd say something like...

```js
    // If we are using the network information API to determine the default
    // bandwidth estimate, use the lower of our bandwidth estimator's estimate
    // and the browser's estimate, for a conservative final estimate.
    // Ignore the browser's estimate if the network information API returns an
    // estimate of 10mbps, however; that is a cap on the output meant to prevent
    // browser fingerprinting, and does not represent a true estimate.
```